templates/zerver/app/index.html

```diff
@@ -68,10 +96,11 @@
 
 <div id="app-loading">
     <div id="app-loading-middle-content">
-        <div id="app-loading-logo">
-            <img src="/static/images/logo/zulip-icon-circle.svg" />
+        <div class="zcontainer">
+            <div class="zspinner"></div>
+            <img class="zlogo" src="https://chat.zulip.org/static/images/logo/zulip-icon-circle.svg"/>
         </div>
-        <h3>{{ _('Loading...') }}</h3>
+        <h3 class="ztext">{{ _('Loading...') }}</h3>
```

Comment:
These class names could be improved -- I think `app-loading-text`, `app-loading-spinner-container`, `app-loading-logo`, and `app-loading-spinner` would be a great set of names for these elements.

(We want to make sure they clearly don't conflict with names used for CSS classes in the app itself).